www/artists/jcowin/js/slideshow.js

```diff
@@ -0,0 +1,28 @@
+var slideIndex = 1;
+showSlides(slideIndex);
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var dots = document.getElementsByClassName("dot");
+  if (n > slides.length) {slideIndex = 1}
+  if (n < 1) {slideIndex = slides.length}
```

Comment:
Future-proofing note: plusSlides allows slideIndex to be changed by an arbitrary amount. This means you may theoretically end up with n more than slides.length + 1, or less than 0. If this occurs, these two lines will move it to the opposite side, but won't properly wrap around. You may want to consider altering the assignment operators to include the difference between n and its limits.